include/quicr/quicr_server_session.h

```diff
@@ -0,0 +1,106 @@
+/*
+ *  quicr_server_session.h
+ *
+ *  Copyright (C) 2023
+ *  Cisco Systems, Inc.
+ *  All Rights Reserved
+ *
+ *  Description:
+ *      This is an interface specification for the session layer sitting at the
+ *      next level of the server library.
+ *
```

Comment:
same comment as ealrier, can we clarify what does next level means

Reply:
Ditto.